package.json

```diff
@@ -16,7 +16,7 @@
   "scripts": {
     "make-dev": "cross-env APP_ENV=dev cross-env APP_DEV_MODE=server webpack-dev-server --config webpack.config.js --history-api-fallback",
     "make-watch": "cross-env APP_ENV=dev cross-env APP_DEV_MODE=watch webpack --watch --config webpack.config.js",
-    "make": "cross-env APP_ENV=production cross-env APP_VERSION=\"git tag --points-at HEAD\" cross-env APP_COMMIT=\"git rev-parse HEAD\" webpack --config webpack.config.js"
+    "make": "cross-env APP_ENV=production webpack --config webpack.config.js"
```

Comment:
`APP_VERSION` and `APP_COMMIT` were broken and will be fixed in #133.
If you have not found other issues with the line than described in #131, I ask you to not change this line.

Reply:
Okay, I've updated this in my fork. But I got an error on build (Windows 7 / 10 64bit):
```shell
C:\nodejs\node.exe C:\nodejs\node_modules\npm\bin\npm-cli.js run make --scripts-prepend-node-path=auto

> klaro@0.2.11 make C:\opensource\klaro
> cross-env APP_ENV=production cross-env APP_VERSION=$(git tag --points-at HEAD) cross-env APP_COMMIT=$(git rev-parse HEAD) webpack --config webpack.config.js

"cross-env" kann syntaktisch an dieser Stelle nicht verarbeitet werden.

C:\projects\opensource\klaro>  "C:\opensource\klaro\node_modules\.bin\\node.exe"  "C:\opensource\klaro\node_modules\.bin\\..\cross-env\src\bin\cross-env.js" APP_ENV=production cross-env APP_VERSION=$(git tag --points-at HEAD) cross
-env APP_COMMIT=$(git rev-parse HEAD) webpack --config webpack.config.js
npm ERR! code ELIFECYCLE
npm ERR! errno 255
npm ERR! klaro@0.2.11 make: `cross-env APP_ENV=production cross-env APP_VERSION=$(git tag --points-at HEAD) cross-env APP_COMMIT=$(git rev-parse HEAD) webpack --config webpack.config.js`
npm ERR! Exit status 255
npm ERR!
npm ERR! Failed at the klaro@0.2.11 make script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\..\AppData\Roaming\npm-cache\_logs\2019-12-05T15_21_50_333Z-debug.log

Process finished with exit code 255
```

```log
===================================
2019-12-05T15_21_50_333Z-debug.log:
===================================
0 info it worked if it ends with ok
1 verbose cli [ 'C:\\nodejs\\node.exe',
1 verbose cli   'C:\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'run',
1 verbose cli   'make',
1 verbose cli   '--scripts-prepend-node-path=auto' ]
2 info using npm@6.5.0-next.0
3 info using node@v11.6.0
4 verbose run-script [ 'premake', 'make', 'postmake' ]
5 info lifecycle klaro@0.2.11~premake: klaro@0.2.11
6 info lifecycle klaro@0.2.11~make: klaro@0.2.11
7 verbose lifecycle klaro@0.2.11~make: unsafe-perm in lifecycle true
8 verbose lifecycle klaro@0.2.11~make: PATH: C:\nodejs\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin;C:\opensource\klaro\node_modules\.bin;C:\nodejs;....;
9 verbose lifecycle klaro@0.2.11~make: CWD: C:\opensource\klaro
10 silly lifecycle klaro@0.2.11~make: Args: [ '/d /s /c',
10 silly lifecycle   'cross-env APP_ENV=production cross-env APP_VERSION=$(git tag --points-at HEAD) cross-env APP_COMMIT=$(git rev-parse HEAD) webpack --config webpack.config.js' ]
11 silly lifecycle klaro@0.2.11~make: Returned: code: 255  signal: null
12 info lifecycle klaro@0.2.11~make: Failed to exec make script
13 verbose stack Error: klaro@0.2.11 make: `cross-env APP_ENV=production cross-env APP_VERSION=$(git tag --points-at HEAD) cross-env APP_COMMIT=$(git rev-parse HEAD) webpack --config webpack.config.js`
13 verbose stack Exit status 255
13 verbose stack     at EventEmitter.<anonymous> (C:\nodejs\node_modules\npm\node_modules\npm-lifecycle\index.js:301:16)
13 verbose stack     at EventEmitter.emit (events.js:188:13)
13 verbose stack     at ChildProcess.<anonymous> (C:\nodejs\node_modules\npm\node_modules\npm-lifecycle\lib\spawn.js:55:14)
13 verbose stack     at ChildProcess.emit (events.js:188:13)
13 verbose stack     at maybeClose (internal/child_process.js:978:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:265:5)
14 verbose pkgid klaro@0.2.11
15 verbose cwd C:\opensource\klaro
16 verbose Windows_NT 6.1.7601
17 verbose argv "C:\\nodejs\\node.exe" "C:\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "make" "--scripts-prepend-node-path=auto"
18 verbose node v11.6.0
19 verbose npm  v6.5.0-next.0
20 error code ELIFECYCLE
21 error errno 255
22 error klaro@0.2.11 make: `cross-env APP_ENV=production cross-env APP_VERSION=$(git tag --points-at HEAD) cross-env APP_COMMIT=$(git rev-parse HEAD) webpack --config webpack.config.js`
22 error Exit status 255
23 error Failed at the klaro@0.2.11 make script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 255, true ]
```